### ISSUE TITLE
summary of squash

### ISSUE THREAD

**Issue by brianjmurrell** [ingydotnet/git-subrepo#349]
It might be useful to at least optionally be able to have a _one line per commit_ summary of the commits that are squashed into a _git subrepo pull_ (maybe clone too, but I could see that getting out of hand) added to the _subrepo_ commit message.

**Comment by brianjmurrell** [comment]
As an example, I am manually adding the following to the commit messages when I pull:

```
    Changes:
    * abcdefg - bla bla bla
    * deadbaf - more bla bla bla
    * 1234567 - blahdy blah bla
```

Now that I think of it, I might just incorporate my manual steps into a githook, but having this right in `git subrepo pull` would be neat.

**Comment by alwaysharsha** [comment]
Yes currently the squash would just have the last commit message in subrepo history and missing all the useful info added into the commit history.

**Comment by FFdhorkin** [comment]
Yeah, this would improve the usefulness of git-subrepo significantly, as it would make traceability a lot easier
